creature/badooga; Better Greatwyrms.json

```diff
@@ -0,0 +1,2138 @@
+{
+	"siteVersion": "1.147.13",
```

Comment:
Remove this `siteVersion` key to pass the tests.

It's automatically added if you download content from the homebrew builder; used to warn the user about using older files in newer versions of the site. It isn't needed for content in the repository.